sqlx-core/src/sqlite/options/connect.rs

```diff
@@ -20,7 +20,19 @@ impl ConnectOptions for SqliteConnectOptions {
             // send an initial sql statement comprised of options
             let mut init = String::new();
 
+            // This is a special case for sqlcipher. When the `key` pragma
+            // is set, we have to make sure it's executed first in order.
+            if let Some(pragma_key_password) = self.pragmas.get("key") {
+                use std::fmt::Write;
```

Comment:
You can import this at the file level which will let you get rid of the import in the `for` loop as well.

Reply:
I usually prefer to tightly scope this import so it's clear which "mode" the `write!()` macro is operating in (since it can use either `std::fmt::Write` or `std::io::Write`) but since there's nothing else going on in this file, I don't think it really matters.

Reply:
Done.